com.unity.render-pipelines.high-definition/Editor/AssemblyInfo.cs

```diff
@@ -1,3 +1,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Unity.RenderPipelines.HighDefinition.Editor.Tests")]
+[assembly: InternalsVisibleTo("Unity.Industrial.Materials.AVRD.Editor")]
```

Comment:
why do you have this change? is it for your test?

Reply:
Already discussed live, see also new PR here https://github.com/Unity-Technologies/Graphics/pull/3988